src/Shared.msbuild

```diff
@@ -6,38 +6,32 @@
     <Authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, Dave Glick, Pascal Berger and contributors</Authors>
     <Company>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, Dave Glick, Pascal Berger and contributors</Company>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageIconUrl>https://raw.githubusercontent.com/cake-build/graphics/master/png/cake-medium.png</PackageIconUrl>
+    <PackageIcon>cake-medium.png</PackageIcon>
```

Comment:
From the experiments that @Jericho has done, I believe that NuGet.org will strip the icon out, as it isn't currently supported there.  Can you confirm @Jericho?

Reply:
.NET Core 3 requires it though or you get an compiler error.
```diff
- error NU5048: The 'PackageIconUrl'/'iconUrl' element is deprecated.
- Learn more at https://aka.ms/deprecateIconUrl
```

Reply:
Hmm, interesting.  So I wonder if there is going to be an update to NuGet.org ahead of the RTM of .Net Core 3.

Reply:
Also, was this previously a warning?  Or has it jumped straight to error?

Reply:
Well currently we treat warnings as errors.

Reply:
Oh, I see.  Ok, so that changes things :smile:

Reply:
I confirm that NuGet.org strops our embedded icons. I uploaded a package with such icon and you can see here that the icon is now missing: https://www.nuget.org/packages/Cake.Email.Common/

Reply:
Looking at NuGet/NuGetGallery/issues/6907 it looks like it's in milestone [S159 - 2019.09.16](https://github.com/NuGet/NuGetGallery/milestone/130) which is due by October 04, 2019 which feels late as .NET Core 3 is due by next week.

Reply:
@Jericho @gep13 Just pushed https://www.nuget.org/packages/LitJson/0.14.1 with PackageIcon and it worked just fine.

Exchanged emails with nuget team and there is no `<icon>` element in nuspec in Cake.Email.Common 0.4.0.
Issue with Cake.Email.Common 0.4.0 is likely not building using .NET Core 3.0 SDK

Can't find any global.json, .NET Core SDK install in build.ps1/build.sh nor appveyor.yml (which seems to use image: Visual Studio 2017), so can't find anything indicating it using newer required SDK.

Reply:
@devlead You're probably right: when I built on my machine (where .net 3.0 is available), the generated package included the icon but when built on [AppVeyor](https://ci.appveyor.com/project/cakecontrib/cake-email-common/builds/27254454/artifacts), the icon was stripped and therefore the package published to NuGet was missing an icon. I incorrectly assumed that NuGet had stripped the icon.

Reply:
That's one use case for having a `global.json` file, so same SDK is used locally and CI.